compiler/ast.nim

```diff
@@ -823,6 +823,7 @@ type
       # check for the owner when touching 'usedGenerics'.
       usedGenerics*: seq[PInstantiation]
       tab*: TStrTable         # interface table for modules
+      nimblePkg*: PSym
```

Comment:
No. We lived without it for a long time so I'm sure it's unnecessary.

Reply:
done, found another way without introducing this field